From b01997fa7986ed31dd11ff8bf9a90930a65936ad Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Mon, 30 Nov 2020 06:00:55 +0100
Subject: [PATCH 341/467] usb: typec: anx7688: Increase wait time for OCM
 messages

When bumping i2c clock, this got too tight for initial FW startup.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 drivers/usb/typec/anx7688.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/typec/anx7688.c b/drivers/usb/typec/anx7688.c
index a8e420e05..89ecdc844 100644
--- a/drivers/usb/typec/anx7688.c
+++ b/drivers/usb/typec/anx7688.c
@@ -331,8 +331,8 @@ static int anx7688_send_ocm_message(struct anx7688 *anx7688, int cmd,
                 dev_err(anx7688->dev,
                         "failed to send pd packet (err=%d)\n", ret);
 
-        // wait until the message is processed (10ms max)
-        for (i = 0; i < 100; i++) {
+        // wait until the message is processed (30ms max)
+        for (i = 0; i < 300; i++) {
                 ret = anx7688_tcpc_reg_read(anx7688, ANX7688_TCPC_REG_INTERFACE_SEND);
                 if (ret <= 0)
                         return ret;
-- 
2.34.0

